gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -369,7 +367,8 @@ def _decrement_device_interface_count(self, device):
         self._update_network_function_device_db(device, device['status'])
 
     def _get_orchestration_driver(self, service_vendor):
-        return self.drivers[service_vendor.lower()]
+        return orchestration_driver.OrchestrationDriver(self.config)
```

Comment:
A new object will be created every time _get_orchestration_driver method is invoked.
Can we just keep this in self.orchestration_driver, and return that here.